src/hooks/usePropsGetters/useSelectInputProps.ts

```diff
@@ -76,39 +78,12 @@ export default function useSelectInputProps(
   }, [selected, currentQuestionData?.id, currentQuestionData?.type, quizAnswerChanged]);
 
   useEffect(() => {
```

Comment:
Can you add a comment here explaining what this effect does and why `answerInputs` is in the dependency array?